File: 0724-find-pivot-index/0724-find-pivot-index.go
func pivotIndex(nums []int) int {
    total := 0
    for _, n := range nums {
        total += n
    }
    
    acc := 0
    for i, n := range nums {
        if (total - n - acc) == acc {
            return i
        }
        acc += n
    }
    return -1
}

File: 0724-find-pivot-index/0724-find-pivot-index.c
int pivotIndex(int* nums, int nums_size){
    int total, acc, i;
    total = acc = 0;
    for (int i = 0; i < nums_size; i++)
        total += nums[i];
    
    for (int i = 0; i < nums_size; i++) {
        if (total - nums[i] - acc == acc)
            return i;
        acc += nums[i];
    }
    return -1;
}